pkg/configtx/organization.go

```diff
@@ -102,18 +99,14 @@ func (c *ConfigTx) ConsortiumOrg(consortiumName, orgName string) (Organization,
 }
 
 // RemoveConsortiumOrg remove an org in a consortium organization from Consortiums groups
-func (c *ConfigTx) RemoveConsortiumOrg(consortiumName, orgName string) error {
-	consortium, ok := c.updated.ChannelGroup.Groups[ConsortiumsGroupKey].Groups[consortiumName]
-	if !ok {
-		return fmt.Errorf("consortium %s does not exist in channel config", consortiumName)
-	}
-	if _, ok := consortium.Groups[orgName]; !ok {
-		return fmt.Errorf("consortium org %s does not exist in channel config", orgName)
-	}
+// Specifying a consortium name or organization name that does not exist in the consortiums ConfigGroup of the channel config will
+// not return a error.
+// Removal will panic if consortiums group does not exist.
```

Comment:
Will also panic if the consortium doesn't exist when attempting to delete from it on line 108